yarn-project/circuits.js/src/cbind/circuits.gen.ts

```diff
@@ -180,55 +180,6 @@ export function fromNativeAggregationState(o: NativeAggregationState): MsgpackNa
   };
 }
 
-interface MsgpackReadRequestMembershipWitness {
-  leaf_index: Buffer;
-  sibling_path: Tuple<Buffer, 32>;
-  is_transient: boolean;
-  hint_to_commitment: Buffer;
-}
```

Comment:
Auto-removed because we are not using msgpack for the cbinds to inner and initial kernels. Msgpack generates this interface and helper functions for serialization only when the struct is serialized as part of a cbind. For now, we are not using msgpack for the private kernel inner/init cbinds, so this was auto-removed. Once we migrate those cbinds to msgpack, this code should be auto-readded.